packages/rulesets/src/spectral/test/xms-pageable-for-list-calls.test.ts

```diff
@@ -206,6 +278,96 @@ test("CollectionObjectPropertiesNaming should find no errors", () => {
             },
           },
         },
+      "{scope}/providers/Microsoft.Music/Configurations": {
+        get: {
+          operationId: "test_ListByID",
+          responses: {
+            200: {
+              description: "Success",
+              schema: {
+                properties: {
+                  value: {
+                    type: "array",
+                  },
+                  nextLink: {
+                    type: "string",
+                  },
+                },
+                required: ["value"],
+              },
+            },
+          },
+          "x-ms-pageable": {
+            nextLinkName: null,
+          },
+        },
+      },
+      "{scope}/providers/Microsoft.Music/Configurations/{configName}": {
+        get: {
+          operationId: "test_ListByID",
+          responses: {
+            200: {
+              description: "Success",
+            },
+          },
+        },
+      },
+      "{scope}/providers/Microsoft.Music/Configurations/config/providers/Microsoft.Song/nestedConfigurations/nestedConfig": {
+        get: {
+          operationId: "test_ListByID",
+          responses: {
+            200: {
+              description: "Success",
+            },
+          },
+        },
+      },
+      "{scope}/providers/Microsoft.Music/Configurations/config/providers/Microsoft.Song/nestedConfigurations/nestedConfig/Microsoft.Voice/deeplyNestedConfigurations/deeplyNestedConfig":
+        {
+          get: {
+            operationId: "test_ListByID",
+            responses: {
+              200: {
+                description: "Success",
+              },
+            },
+          },
+        },
+      "/providers/Microsoft.ConnectedVMwarevSphere/virtualMachiness": {
+        get: {
+          operationId: "Good_List",
+          responses: {
+            200: {
+              description: "Success",
+              schema: {
+                properties: {
+                  value: {
+                    type: "array",
+                  },
+                  nextLink: {
+                    type: "string",
+                  },
+                },
+                required: ["value"],
+              },
+            },
+          },
+          "x-ms-pageable": {
+            nextLinkName: null,
+          },
+        },
+      },
+      "/{resourceUri}/providers/Microsoft.ConnectedVMwarevSphere/virtualMachines/virtualMachinesInstances/nestedvirtualMachines/nestedvirtualMachinesInstances":
```

Comment:
nit: swaggers usually have parameters for resource names
```suggestion
      "/{resourceUri}/providers/Microsoft.ConnectedVMwarevSphere/virtualMachines/{virtualMachinesInstances}/nestedvirtualMachines/{nestedvirtualMachinesInstances}":
```